infrastructure/php-container/src/.htaccess

```diff
@@ -19,21 +19,21 @@ DirectorySlash off
     RewriteRule ^graphql$ api/public/graphql [L]
 
     # Send admin requests to admin public folder; don't rewrite twice.
-    RewriteCond %{REQUEST_URI} !^admin/public/
-    RewriteRule ^admin(/(.*))?$ admin/public/$2 [L]
+    RewriteCond %{REQUEST_URI} !^frontend/admin/public/
+    RewriteRule ^admin(/(.*))?$ frontend/admin/public/$2 [L]
```

Comment:
Okay, I've wrapped my head around the /public issue. Requests to `/admin/public/.*` previously didn't need any redirection in htaccess, but now they do.

https://github.com/GCTC-NTGC/gc-digital-talent/commit/5fe15a29c6fd576387612575a9121f7f97205f55 could be reverted if we instead changed these lines to:
```
    # Send admin requests to admin public folder
    RewriteRule ^admin/public(/(.*))?$ frontend/admin/public/$2 [L]
    RewriteRule ^admin(/(.*))?$ frontend/admin/public/$2 [L]
```

Reply:
Ok will do!

Reply:
Storybook images will still be broken links, but can resolve that in another PR